client/a8c-for-agencies/sections/sites/sites-header-actions/index.tsx

```diff
@@ -69,15 +69,12 @@ export default function SitesHeaderActions( { onWPCOMImport }: Props ) {
 					onCreateSiteSuccess={ onCreateSiteSuccess }
 				/>
 			) }
-			{ devSitesEnabled && (
+			<div ref={ ( ref ) => setTourStepRef( ref ) }>
 				<AddNewSiteButton
 					showMainButtonLabel={ ! isMobile }
-					devSite
+					onWPCOMImport={ onWPCOMImport }
```

Comment:
GitHub is not highlighting the change very well here.

What's actually being change here is that the following code is removed completely (first `AddNewSiteButton` component):

```jsx
{ devSitesEnabled && (
				<AddNewSiteButton
					showMainButtonLabel={ ! isMobile }
					devSite
					toggleDevSiteConfigurationsModal={ toggleDevSiteConfigurationsModal }
				/>
			) }
```

and the `toggleDevSiteConfigurationsModal={ toggleDevSiteConfigurationsModal }` prop is now passed to the second `AddNewSiteButton` component.